common/script/script-api/src/main/java/org/thingsboard/script/api/tbel/TbDate.java

```diff
@@ -218,9 +218,12 @@ private DateTimeFormatOptions getDateFormattingOptions(String optionsStr) {
         return opt;
     }
 
-    public long now() {
+    public static long now() {
         return Instant.now().toEpochMilli();
     }
+    public static String nowToString() {
+        return Instant.now().toString();
+    }
```

Comment:
Do we really need this method?..
Users could use `new Date().toString()`.

Reply:
Ok